src/composables/search/search-filters.ts

```diff
@@ -0,0 +1,46 @@
+import { ref } from 'vue'
+export const initialFilters = {
+  layers: true,
+  cms: true,
+  address: false,
+  parcels: false,
+  localite: false,
+  lieudit: false,
+  commune: false,
+  flik: false,
+  hydro: false,
+  biotopes: false,
+  extent: false,
+  activeLayers: true,
+}
+
+export const curFilters = ref(
+  Object.assign(
+    {},
+    localStorage.getItem('searchFacets')
```

Comment:
Usually we would use the state persistor logic for this (with only one reader, the local storage), but i let you judge if it is worth the change.